src/test/test_ISFirmwareUpdate.cpp

```diff
@@ -430,7 +431,8 @@ TEST(ISFirmwareUpdate, pack_unpack__req_update)
     EXPECT_EQ(*(uint32_t*)(buffer+0), fwUpdate::TARGET_GPX1);       // uint32_t (0 + 4)
     EXPECT_EQ(*(uint32_t*)(buffer+4), fwUpdate::MSG_REQ_UPDATE);    // uint32_t (4 + 4)
     EXPECT_EQ(*(uint16_t*)(buffer+8), session_id);                  // uint16_t (8 + 2)
-    EXPECT_EQ(*(uint16_t*)(buffer+10), 1);                          // uint16_t (10 + 2)
+    EXPECT_EQ(*(uint8_t*)(buffer+10), 1);                          // uint16_t (10 + 1)
```

Comment:
Update the comments to reflect uint8_t, and make sure the maths are still good and columns are aligned.